client/src/components/admin/AdminTShirt.vue

```diff
@@ -71,21 +75,54 @@
   </div>
 </template>
 
-<script setup>
+<script setup lang="ts">
 import ApiService from "@/services/ApiService";
 import { ref } from "vue";
 import { useRouter } from "vue-router";
+import type { Ref } from "vue";
+import type { UserData } from "@/types/UserData";
+
+interface TShirtStat {
+  tshirtSize: string;
+  gender: string;
+  amount: number;
+}
 
 const router = useRouter();
 
-const entries = ref([]);
-const stats = ref([]);
+const entries: Ref<UserData[]> = ref([]);
+const stats: Ref<TShirtStat[]> = ref([]);
 
 const onExport = () => {
-  //TODO: Implement an CSV export
-  alert(
-    "Diese Funktion ist noch nicht implementiert! Wir sollten zuerst absprechen, welche Daten wirklich alle benötigt werden"
+  const propsToExport = [
+    "club",
+    "fullName",
+    "tshirtSize",
+    "gender",
+    "email",
+    "address",
```

Comment:
Die Adresse sollten wir besser auch in einzelne Spalten aufteilen, JSON im csv macht glaube ich nicht so viel Sinn.